frontend/components/item/util/ViewTextLang.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <span v-if="value && value.value">
+  <span v-if="value && value.value && contentView">
     <template v-if="value.pbid">
       <NuxtLink :to="getUrlFromPBID(value.item)">
-        <!-- eslint-disable-next-line vue/no-v-html -->
-        <span :title="tooltip" v-html="value.value" />
+        <span :title="tooltip" v-html="contentView" />
       </NuxtLink>
     </template>
     <template v-else>
-      <!-- eslint-disable-next-line vue/no-v-html -->
```

Comment:
Please enable _eslint_ plugin in you local development and recover this line to avoid a _eslint_ warning.